README.md

```diff
@@ -426,6 +426,8 @@ where:
      if full primary key equal condition is specified
   * `timeout` (`?number`) - `vshard.call` timeout (in seconds)
   * `fields` (`?table`) - field names for getting only a subset of fields
+  * `fullscan` (`?boolean`) - if `true` then a critical log entry will be skipped
+    on potentially long `count`, see [avoiding full scan](https://github.com/tarantool/crud/blob/master/doc/select.md#avoiding-full-scan).
```

Comment:
In my opinion, you mixed up the place of correction, this description was correct ('...on potentially `select`...'), the correction had to be done in `count` section which is below.

Reply:
Yes, I'm sorry. I hope it is correct now.

Reply:
It's correct now, thank you!